tests/monotouch-test/CoreAnimation/EmitterBehaviorTest.cs

```diff
@@ -23,6 +23,7 @@
 
 namespace MonoTouchFixtures.CoreAnimation {
 
+	[Ignore ("CAEmitterBehavior is now being rejected by Apple")]
```

Comment:
Maybe mention the radar again because I was about to comment: "why not remove?" (: